### PR TITLE
run_docker start - specify argument from environment variables

### LINKED ISSUE #1330: Cannot specify inbound transport through environment variable
At present, I don't think it is possible to specify `--inbound-transport` through the accompanying environment variable `ACAPY_INBOUND_TRANSPORT`. This stems from this argument being both an "append" and "nargs > 1" argument. The ConfigArgParse library does not accept a list of lists at the current version in use by ACA-Py or in the most recent release (1.5.1 at the time of writing).

I have a PR to ConfigArgParse that should address this: https://github.com/bw2/ConfigArgParse/pull/240

Until my or another fix is merged and released, I figured it made sense to call out this limitation here.

If my assessment is off, I'd be happy to hear how you did it as all permutations I attempted and time spent examining the code seems to indicate this is just broken in ConfigArgParse lol.

### PR DESCRIPTION
**Pull request by shaangill025**
Signed-off-by: Shaanjot Gill <shaangill025@users.noreply.github.com>
- resolve #1330 
- @MonolithicMonk I think you are starting ACA-Py agent from `./scripts/run_docker start ...`. You can either specify the environment variables in `./scripts/run_docker` by adding each one of them as something like:
```
$CONTAINER_RUNTIME run --rm -ti --name "aries-cloudagent-runner_${RAND_NAME}" \
-e ACAPY_OUTBOUND_TRANSPORT=http -e ACAPY_INBOUND_TRANSPORT=[[\"http\",\"0.0.0.0\",\"8021\"],[\"ws\",\"0.0.0.0\",\"8023\"]] \
    $ARGS aries-cloudagent-run "$@"
```
- Another option [if this PR is approved and merged], is just to export the `environment variable` as usual. Updated script will go through all the `environment variables`, extract all with `*ACAPY_*` and `*=*` pattern match and add them to `docker run ... -e env_var -e env_var .....`.

### CODE REVIEW

File: scripts/run_docker [hunk 1]
```diff
@@ -14,6 +14,12 @@ for PORT in $PORTS; do
   ARGS="${ARGS} -p $PORT"
 done
 
+ENV_VARS=$(printenv)
```
[the remark this answers is not in the document]
Comment (shaangill025) [reply scripts/run_docker:17]: Sounds good, can I please get your feedback on another approach I had initially implemented. Basically environment variables to be specified in a similar manner to `PORTS`, I think this provides more direct control and can prevent inadvertent mistakes/issues:
```
PORTS="5004 8004" ENV_VARS="ACAPY_OUTBOUND_TRANSPORT=http ACAPY_INBOUND_TRANSPORT=[[\"http\",\"0.0.0.0\",\"8021\"],[\"ws\",\"0.0.0.0\",\"8023\"]]" ./scripts/run_docker start ...
```

**Comment by MonolithicMonk** [comment]
@shaangill025 I am running the agent image using a kubernetes deployment yaml file.  Setting the `ACAPY_INBOUND_TRANSPORT` environment variable using the nested method results in error message: `expected 3 arguments`

**Comment by shaangill025** [comment]
> @shaangill025 I am running the agent image using a kubernetes deployment yaml file. Setting the `ACAPY_INBOUND_TRANSPORT` environment variable using the nested list method results in error message: `expected 3 arguments`

It shouldn't be specified as `ACAPY_INBOUND_TRANSPORT=[["http","0.0.0.0","8021"]]` but as `ACAPY_INBOUND_TRANSPORT=[[\"http\",\"0.0.0.0\",\"8021\"]]`

**Comment by WadeBarnes** [comment]
@shaangill025, When you have things the way you want them could you rebase the changes to clean up the commit history?

**Comment by shaangill025** [comment]
Based upon feedback from @ianco and @WadeBarnes , I think the explicit approach of using `ENV_VARS` is a better approach. Extracting all environment variables can lead to inadvertent mistakes as unwanted variables will need to be `unset` and updated variables need to be re-exported when starting up ACA-Py agents with different config. 
Also, semantically `PORTS` [`-p`] and environment variables [`-e`] for the `aries-cloudagent-runner_...` docker container are similar, so `ENV_VARS` makes sense.
@MonolithicMonk for your reference, the start command will be like:
```
PORTS="5002 8002" ENV_VARS="ACAPY_INBOUND_TRANSPORT=[[\"http\",\"0.0.0.0\",\"8021\"]]" ./scripts/run_docker start ......
```

**Pull request closed without merging**